services/web-app/package.json

```diff
@@ -59,5 +62,16 @@
     "sass": "^1.49.9",
     "slugify": "^1.6.5",
     "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@storybook/addon-actions": "^6.4.22",
+    "@storybook/addon-essentials": "^6.4.22",
+    "@storybook/addon-links": "^6.4.22",
+    "@storybook/addon-postcss": "^2.0.0",
+    "@storybook/builder-webpack5": "^6.4.22",
+    "@storybook/manager-webpack5": "^6.4.22",
+    "@storybook/react": "^6.4.22",
+    "sass-loader": "^12.6.0",
```

Comment:
is this dependency only needed to build storybook?

Reply:
Yes, `sass-loader` needed for Storybook webpack.